bin/run_generic_job.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+SUFFIX=$1
+shift
+ARGS=$(perl -e 'print "[ \"", join("\", \"",@ARGV), "\"]"' "$@")
+kubectl -n holdings create -f - <<EOT
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: holdings-client-$SUFFIX
```

Comment:
I tend to use the ${} syntax in these embedded situations; maybe superstition.

Reply:
:+1: